homework-submission/react-hyf/react-hyf-client/src/components/week3/UserInfo/index.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+function UserInfo({ userValues }) {
+  const getNameStyle = {
+    fontSize: '2.5rem',
+    fontWeight: 'bold',
+  };
+  return (
+    <React.Fragment>
+      <p style={getNameStyle}>{` ${userValues[0]} ${userValues[1]}`}</p>
+      <div>
+        <p> {`👤 ${userValues[2]}`}</p>
+        <p> {`🌐 ${userValues[3]}`}</p>

```

Comment:
Please show more values you get from the API as requested in the homework and shown here: https://uinames.com/

Reply:
I misunderstood the homework . https://uinames.com/api/?amount=10 What I see in this link is only those information. I added picture because someone asked in Slack. So I am going to add the others.